### ISSUE TITLE
Implement configurable learning rate scheduler

### ISSUE THREAD

**Issue by drewoldag** [lincc-frameworks/hyrax#135]
Learning Rate scheduler ([described here](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate)) to be defined in the config.

Also note that learning rate schedulers can be "stacked" as in the second example code in the link listed above. This increases the complexity of the problem both in terms of the way we define things in the config file and how we implement the scheduler in the code. 

@aritraghsh09 has already mentioned that indeed people do tend to use LR schedulers in the real world, but, IIRC, he hasn't yet used them. 

Finally - if we do implement configurable LR schedulers, we should add the LR value to the tensorboard output.
